apps/financial-aid/backend/migrations/20220303095125-applicaitons-add-column-applicationid.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.addColumn(
+          'applications',
+          'application_id',
+          {
+            type: Sequelize.UUID,
+            allowNull: true,
+            unique: true,
```

Comment:
unique: true this should be false, because there can come a case where the applicant is able to create application in our DB but it didn't go through in the Application System DB